dql/history.py

```diff
@@ -55,3 +55,25 @@ def try_to_write_history(self, history_dir: Optional[str] = None) -> None:
             else:
                 # append will fail if the file does not exist.
                 readline.append_history_file(new_history_length, history_file)
+
+    def remove_items(self, n=1):
+        """Remove items from current session's in-memory history."""
+        if n <= 0:
+            return
+
+        try:
+            import readline
+        except ImportError:
+            # Windows doesn't have readline, so gracefully ignore.
+            pass
```

Comment:
This should be a `return`, otherwise it will crash on the next line when you use readline

Reply:
Good catch!